### ISSUE TITLE
Can I ask what that means and how to fix it?

### ISSUE THREAD

**Issue by xuerange** [xiangruili/dicm2nii#61]
Sag_DKI_25dir_2B: bvec sign for non-axial acquisition with ROW phase direction not tested.
 Please check the result and report problem to author.

**Comment by neurolabusc** [comment]
@xiangruili this seems timely. You may want to see the GE validation dataset and dcm2niix bvecs with [this PR](https://github.com/rordenlab/dcm2niix/pull/736).

**Comment by mr-jaemin** [comment]
I tested non-axial acquisitions (with dcm2niix) and didn't experience any issue so far. Here is some validation dataset:

[exam5382_multishell_orientation_invivo (1.6 GB)](https://gehealthcare-amer.my.salesforce.com/sfc/p/30000001F94O/a/3a000000YJy3/fQmAcuFDbgf.AQuh4Ic2oOCPK4sVrlDNX4nXaocde5Q)
See README_multishell_DTI_Orientations_Ex5382.pptx
Axial/Sag/Cor various orientation including
Strange oblique suggested [by Chris](https://people.cas.sc.edu/rorden/mricron/bvec.doc)

As @neurolabusc noted above, I found that feet first DTI works the same way as head first.

**Comment by xuerange** [comment]
Hello, I asked this question on GitHub "Can I ask what that means and how to fix it?" and I saw your prompt reply.

I tested the sagittal DKI with GE scan and got an error. Here are a few examples of the data, can you help me see what the problem is? Thank you very much for your reply!
















At 2023-08-08 20:44:39, "xiangruili" ***@***.***> wrote:

This should be a diffusion scan from GE, right? Most people use axial slices, so that is well-tested. For non-axial slices, we have very limited testing dataset, so the sign of the bvec might be wrong for GE system. You can test the result, let us if it is right, or you can share the dataset with us, the converter developers, so we can test it.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you authored the thread.Message ID: ***@***.***>

从网易163邮箱发来的超大附件推荐客户端极速下载
GE_SagDKI.rar (487.4M, 2023年8月24日 11:37 到期)
下载

**Comment by xuerange** [comment]
@xiangruili Thank you very much for your reply, I tried to send you the data yesterday but it didn't work, I don't know how to upload the data, I had compressed the data and tried to drag it directly to the dialog box but it prompted me that the file was too big to upload!

**Comment by xuerange** [comment]
@xiangruili May I ask Prof. Lee to give me an e-mail address?

**Comment by xiangruili** [comment]
@neurolabusc 
I just compared the bvec from dicm2nii.m and dcm2niiX (v1.0.20220720). The FF version gives different bvec, but I think I was using older version of dcm2niiX. In dicm2nii.m, I don't do anything differently for HF vs FF, and that is what you verified, right?

**Comment by mr-jaemin** [comment]
@xiangruili Yes, the bvec conversion for FF would be same as HF based on the validation dataset.
